indico_toolkit/indico_wrapper/indico_wrapper.py

```diff
@@ -1,18 +1,50 @@
-from typing import List, Dict, Union
+from typing import List, Union
 from indico.queries import (
     RetrieveStorageObject,
     GraphQLRequest,
     JobStatus,
     CreateModelGroup,
     ModelGroupPredict,
+    CreateStorageURLs
 )
 from indico.types import Dataset, ModelGroup
 from indico import IndicoClient
+from indico.errors import IndicoRequestError
+from functools import wraps
+import time
 
 from indico_toolkit.types import Predictions
 from indico_toolkit import ToolkitStatusError
 
 
+def retry(exceptions, wait=0.5):
```

Comment:
I kind of think that this function should be in its own file at the same level as client.py and errors.py "retry.py"